docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -11,7 +11,7 @@ su -l
 Zonemaster::Backend requires new dependencies. Depending on the used OS, run
 the corresponding command.
 
-### Centos
+### Rocky Linux
```

Comment:
Is it really meaningful to have Rocky Linux here when Rocky Linux was not supported before?

Reply:
I see your point, I am unsure what to put here as Centos is also invalid as not supported anymore.

Reply:
I guess it could be left  empty since this is the first time there are instructions for Rocky Linux.